dotcom-rendering/src/web/components/FrontCard.tsx

```diff
@@ -0,0 +1,50 @@
+import { ArticleDesign } from '@guardian/libs';
+import type { Props as CardProps } from './Card/Card';
+import { Card } from './Card/Card';
+
+type Props = {
+	trail: TrailType;
+} & Partial<CardProps>;
+
+/**
+ * A wrapper around the normal Card component providing sensible defaults for Cards on front containers.
+ *
+ * Any prop used by the original Card can be used in FrontCard to override the defaults.
+ *
+ * Note: Below parameters are not an exhaustive list of params used by FrontCard, rather they are a list of
+ * commonly overridden params to make it easier for a developer to know if they actually need to override these values.
+ *
+ * @param {SmallHeadlineSize} [headlineSize="medium"] - Defaults to "medium"
+ * @param {ImagePositionType} [imagePosition="top"] - Defaults to "top"
+ * @param {ImagePositionType} [imagePositionOnMobile="left"] - Defaults to "left"
+ * @param {ImageSizeType} [imageSize="medium"] - Defaults to "medium"
+ * @param {DCRSupportingContent[]} [supportingContent=undefined] - Defaults to undefined, set to trail.supportingContent if you want this card to show sublinks.
+ */
+export const FrontCard = (props: Props) => {
+	const { trail } = props;
```

Comment:
If I'm following this correctly, then `props` still has the `trail` key on line 49 when it gets passed to `Card`. But the `Props` type for `Card` doesn't have a `trail` key. So this is a small point but I feel like it could be a bit confusing that we're passing this prop that won't be used?

@mxdvl 's [recent comment](https://github.com/guardian/dotcom-rendering/pull/5383/commits/5bcae7d406cb7fb03f90d1a21e71a7cbd507da17#r920304759) reminded me that we can spread within a destructuring assignment, so we could possibly do something like:

```const { trail, ...rest } = props;```

here, and then just pass `{ ...defaultProps, ...rest }` on line 49? I don't think this is a big deal either way though :)

Reply:
Fancy! Thanks for the tip, I'll give it a go!